App/Components/_appColors.js

```diff
@@ -0,0 +1,19 @@
+import colors from './_colors';
+
+const appColors = {
```

Comment:
Having separate keys give us more granular control over the colors. For example, `backgroundColorQuestion` and `backgroundColorMenuItem` share the same shade of blue. We could technically use something like `backgroundColorBlue` for both elements, but if down the road we decide to have different colors for these elements, we'd have to go back to each file and change the color. If we have separate keys for controlling the color of the element, we can just tweak the relevant keys in `_appColors`.